interview-questions.md

```diff
@@ -8,62 +8,68 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
-
-  Researched answer:
-
+  Your answer: A parameter is a variable meaning it can be called upon whereas an argument is a value that is passed within function.
 
+  Researched answer: After research i found that a argument is a value that is passed through or within a function WHEN the function is called upon. A parameter is used and referred to DURING a function. 
+//
+//
+//
+//
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-  Your answer:
-
-  Researched answer:
-
+  Your answer: The .map() method takes in two or more parameters. I believe that the first parameter starts with the array that is being mapped because .map() only works on arrays and the second or more parameter is optional and can be a function or a value if required. 
 
+  Researched answer: .map predefined parameters are the array that is being mapped and the function or value that is being mapped to the array. The first parameter is the array that is being mapped and the second parameter is the function or value that is being mapped to the array. The second parameter is optional and can be a function or a value if required.
+//
+//
+//
+//
 
 3. What is the difference between map and filter?
 
-  Your answer:
-
-  Researched answer:
-
+  Your answer: Map is used to look through the whole array while filter is used to look through an array but return only certain parts hence "string"
 
+  Researched answer: I think looking up the answer confused me more now but from my understanding the return that you get from .map() is with changes to the original array where the return that you get from .filter() is with no changes to the original array
```

Comment:
It is important to note that the original array is not mutated in either scenario.  Map will return a new array the same length as the original, and filter will return a new array that is a subset of the original array based on a condition.